apps/prairielearn/src/batched-migrations/20240108185602_group_users__group_config_id__backfill.sql

```diff
@@ -5,6 +5,6 @@ SET
 FROM
   groups AS g
 WHERE
-  gu.group_id = g.id
-  AND gu.group_id >= $min
-  AND gu.group_id <= $max;
+  gu.team_id = g.id
+  AND gu.team_id >= $min
+  AND gu.team_id <= $max;
```

Comment:
I don't think it's a good idea to update old migrations...

Reply:
Yes 😄

Reply:
Fixed in 101bab5d52c956552a6c43fbed362b0013ab7f61.